rollup.config.js

```diff
@@ -38,6 +40,10 @@ export default {
       format: 'esm',
       sourcemap: true,
     },
+    {
+      file: "dist/index.js",
+      format: "esm"
+    },
```

Comment:
do we need this block ?

Reply:
Indeed we do not. I did it because the documentation said so. But I tried without it and it works fine. Just deleted it